.ci/ansible/tasks/install_deps.yml

```diff
@@ -51,3 +51,19 @@
 - name: Install ssh-import-id python package to copy public SSH keys from Github accounts
   pip:
     name: ssh-import-id
+
+- name: Install dependencies (MacOS)
```

Comment:
We probably need this task to me moved before L51, so that pip is able to install `ssh-import-id`